test/use-swr-integration.test.tsx

```diff
@@ -28,7 +28,10 @@ describe('useSWR', () => {
 
   it('should allow functions as key and reuse the cache', async () => {
     function Page() {
-      const { data } = useSWR(() => sharedKey, () => 'SWR')
+      const { data } = useSWR(
```

Comment:
maybe set the width to larger to keep them as before?

Reply:
I think 80-char is the standard though.